jedi.inc

```diff
@@ -1590,6 +1590,10 @@
     {$DEFINE SUPPORTS_OUTPARAMS}
     {$DEFINE SUPPORTS_WIDECHAR}
     {$DEFINE SUPPORTS_WIDESTRING}
+    {$IFDEF CPU64}
+		{$DEFINE USE_64BIT_TYPES}
+	{$ENDIF}
+	{$IFDEF HASINTF}
```

Comment:
The correct indentation is two spaces, not four, not tabs
And the last `IFDEF` will most likely render the file unusable as it is not balanced with a closing `ENDIF`

Reply:
If I had to guess the check for HASINTF is a merge remnant from before my changes at the end of December where I moved the check for the HASINTF define into a check for FPC 2.0 and 2.1.

Reply:
- `USE_64BIT_TYPES` is used only in `JclPeImage.pas` at the 6631 line to define the same function `InternalReadProcMem` with a `Size: SIZE_T` for 64-bit and a `Size: Integer` for 32-bit. Seems that this conditional has been defined solely for the purpose of defining the `InternalReadProcMem` function;

- Looking at how `SIZE_T` is defined for 64-bit as a `QWord` and for 32-bit as a `DWord`, the `USE_64BIT_TYPES` conditional could be left aside and removed and define the `InternalReadProcMem` function having the problematic parameter simply defined as `Size: SIZE_T`;

- `{$IFDEF HASINTF}` is an inattention error